portals/knowledge-management/src/Components/Header.vue

```diff
@@ -158,11 +158,7 @@
                     </div>
 
                     <div class="flex items-center gap-4">
-                        <GlobalSearchBar
-                            v-if="
-                                route.name === 'view-article' || route.name === 'services' || route.name === 'licenses'
-                            "
-                        ></GlobalSearchBar>
+                        <GlobalSearchBar v-if="!['home', 'view-category'].includes(route.name)" />
```

Comment:
Should we consider making this more explicit with like a prop that defaults to true for this to display and the page that loads it can decide if it wants to load this or not?